IdentityCore/src/broker_operation/request/token_request/MSIDBrokerOperationTokenRequest.m

```diff
@@ -77,7 +118,14 @@ - (NSDictionary *)jsonDictionary
     if (!configurationJson) return nil;
     [json addEntriesFromDictionary:configurationJson];
     json[MSID_PROVIDER_TYPE_JSON_KEY] = MSIDProviderTypeToString(self.providerType);
-    
+    json[MSID_BROKER_EXTRA_OIDC_SCOPES_KEY] = self.oidcScope;
+    json[MSID_BROKER_EXTRA_QUERY_PARAM_KEY] = self.extraQueryParameters;
+    json[MSID_BROKER_INSTANCE_AWARE_KEY] = [@(self.instanceAware) stringValue];
+    json[MSID_BROKER_INTUNE_ENROLLMENT_IDS_KEY] = [self.enrollmentIds msidJSONSerializeWithContext:nil];
+    json[MSID_BROKER_INTUNE_MAM_RESOURCE_KEY] = [self.mamResources msidJSONSerializeWithContext:nil];
+    json[MSID_BROKER_CLIENT_CAPABILITIES_KEY] = [self.clientCapabilities componentsJoinedByString:@","];
```

Comment:
would it be safer to url encode this? This is technically any random string from the app side.

Reply:
URLQueryItem will url encode all params.